### PR TITLE
General bug fixes including support for original Escher options such as full_screen as well as minor linting issues. Also updated fonts.

### PR DESCRIPTION
**Pull request by spaghettikinase**

### CODE REVIEW

**Comment by zakandrewking** [comment]
what's the status of this PR>

**Comment by zakandrewking** [comment]
These fixes look great. Can you provide a short description of the PR above?

File: src/Builder.js [hunk 1]
```diff
@@ -262,16 +261,21 @@ class Builder {
       // Set up quick jump
       this._setup_quick_jump(this.selection)
 
-      this.callback_manager.run('first_load', this)
```
Comment (zakandrewking) [inline src/Builder.js]: Here's the bug

File: src/Builder.js [hunk 2]
```diff
@@ -262,16 +261,21 @@ class Builder {
...
-
       if (message_fn !== null) setTimeout(message_fn, 500)
     })
+    var s = this.selection
```
Comment (zakandrewking) [inline src/Builder.js]: This should go before first_load

Comment (zakandrewking) [reply src/Builder.js]: Probably before line 223

Comment (spaghettikinase) [reply src/Builder.js]: Done

**Pull request merged**